src/apps/inventory/views.py

```diff
@@ -21,6 +21,12 @@ def get_queryset(self):
         queryset = Product.objects.filter(category=category, is_active=True)
         return queryset
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        images = self.object.images.filter(is_active=True)
+        context["product_images"] = images
```

Comment:
Can you explain why do we need product images

Reply:
Well, still, wy do we need produkt images in context? Do you use product_images value in templates?

Reply:
Previously I wanted to get exactly active images to from the db and pass it to templates. In templates i was using product_images. Currently I'm doing basically the same way but with different approach

Reply:
So, do you still need it?